cmd/agent/app/run.go

```diff
@@ -479,6 +483,11 @@ func StopAgent() {
 	// gracefully shut down any component
 	common.MainCtxCancel()
 
+	if common.ExpvarServer != nil {
+		if err := common.ExpvarServer.Shutdown(context.TODO()); err != nil {
+			log.Errorf("Error shutdown expvar server: %v", err)
+		}
+	}
```

Comment:
We should probably have a shutdown context, with a timeout, rather than TODO.  We've had a few issues in release lately where shutdown just hangs.

In any case, if this isn't a TODO thing, let's change that to Background.

Reply:
Thank you for your review!
https://github.com/DataDog/datadog-agent/pull/11023/commits/f6e82d68b5d928ac163c42bd793c2a9233216e39